src/api/posts/get/getPosts.js

```diff
@@ -1,15 +1,25 @@
+//imports
```

Comment:
Decomposition is good way to control your  app logic, BUT...I suppose that you can create posts.api and add logic methods in there (get, post, patch, puth)

'/api/posts/get/getPosts.js' is too complicated way for this app